djtriggers/models.py

```diff
@@ -103,7 +103,8 @@ def process(self, force=False, logger=None, dictionary=None, use_statsd=False):
         # The lock assures no two tasks can process a trigger simultaneously.
         # The check for date_processed assures a trigger is not executed multiple times.
         try:
-            with NonBlockingLock.objects.acquire_lock(self):
+            with Lock(redis=Redis.from_url(settings.CELERY_BROKER_URL), name='djtriggers-' + str(self.id),
```

Comment:
Why do you use `CELERY_BROKER_URL` for this? I'm okay with Django's cache, but this seems like a stretch.

Reply:
Introduced a new setting for it.